cellpack/autopack/upy/simularium/simularium_helper.py

```diff
@@ -1348,3 +1354,38 @@ def raycast(self, **kw):
 
     def raycast_test(self, obj, start, end, length, **kw):
         return
+
+    def post_and_open_file(self, file_name):
+        simularium_file = Path(f"{file_name}.simularium")
+        url = None
+        try:
+            url = simulariumHelper.store_results_to_s3(simularium_file)
+        except Exception as e:
+            aws_readme_url = "https://github.com/mesoscope/cellpack/blob/feature/main/README.md#aws-s3"
+            if isinstance(e, NoCredentialsError):
+                print(
+                    f"need to configure your aws account, find instructions here: {aws_readme_url}"
+                )
+            else:
+                print(
+                    f"An error occurred while storing the file {simularium_file} to S3: {e}"
+                )
+        if url is not None:
+            simulariumHelper.open_in_simularium(url)
+
+    @staticmethod
+    def store_results_to_s3(file_path):
```

Comment:
I might consider moving more of this into the handler. so in this file all you have to do is call `url = AWSHandler(OPTIONS).upload_file(file_path)` and everything is handled by the other file